src/pysource/models.py

```diff
@@ -63,28 +63,26 @@ def initialize_damp(damp, padsizes, spacing, fs=False):
         mask => 1 inside the domain and decreases in the layer
         not mask => 0 inside the domain and increase in the layer
     """
-    lqmin = np.log(.1)
-    lqmax = np.log(100)
-    w0 = 1/(10 * np.max(spacing))
-
-    z = damp.dimensions[-1]
-    eqs = [Eq(damp, 1)]
-
+    eqs = [Eq(damp, 1.0 if abc_type == "mask" else 0.0)]
     for (nbl, nbr), d in zip(padsizes, damp.dimensions):
-        if not fs or d is not z:
-            nbl = max(5, nbl - 10) if d is z else nbl
+        if not fs or d is not damp.dimensions[-1]:
+            dampcoeff = 1.5 * np.log(1.0 / 0.001) / (nbl)
```

Comment:
appreciate a reference

Reply:
Same, reverting check older versions. SAme as the one in Devito is a standard ABC